docs/loading/Spark-connector-starrocks.md

```diff
@@ -75,41 +85,64 @@ Directly download the corresponding version of the Spark connector JAR from the
-| StringType      | STRING              |
-| DateType        | DATE                |
-| TimestampType   | DATETIME            |
+- The default data type mapping is as follows:
+
+| Spark data type | StarRocks data type                                          |
+| --------------- | ------------------------------------------------------------ |
+| BooleanType     | BOOLEAN                                                      |
+| ByteType        | TINYINT                                                      |
+| ShortType       | SMALLINT                                                     |
+| IntegerType     | INT                                                          |
+| LongType        | BIGINT                                                       |
+| StringType      | LARGEINT                                                     |
+| FloatType       | FLOAT                                                        |
+| DoubleType      | DOUBLE                                                       |
+| DecimalType     | DECIMAL                                                      |
+| StringType      | CHAR                                                         |
+| StringType      | VARCHAR                                                      |
+| StringType      | STRING                                                       |
+| DateType        | DATE                                                         |
+| TimestampType   | DATETIME                                                     |
+| ArrayType       | ARRAY <br> **NOTE:** <br> **Supported since version 1.1.1**. For detailed steps, see [Load data into columns of ARRAY type](#load-data-into-columns-of-array-type). |
+
+- You can also customize the data type mapping.
+
+  For example, a StarRocks table consists of the BITMAP and HLL data types, but Spark does not support the two data types. You need to customize the corresponding data types in Spark. For detailed steps, see load data into columns of [BITMAP](#load-data-into-columns-of-bitmap-type) and [HLL](#load-data-into-columns-of-hll-type) types. **BITMAP and HLL are supported since version 1.1.1**.
+
+## Upgrade Spark connector
+
+### Upgrade from version 1.1.0 to 1.1.1
+
+- Since 1.1.1, the Spark connector does not provide `mysql-connector-java` which is the official JDBC driver for MySQL, because of the limitations of the GPL license used by `mysql-connector-java`.
+  However, the Spark connector still needs the MySQL JDBC driver to connect to StarRocks for the table metadata, so you need to add the driver to the Spark classpath manually. You can find the
+  driver on [MySQL site](https://dev.mysql.com/downloads/connector/j/) or [Maven Central](https://repo1.maven.org/maven2/mysql/mysql-connector-java/).
+- Since 1.1.1, the connector uses Stream Load interface by default rather than Stream Load transaction interface in version 1.1.0. If you still want to use Stream Load transaction interface, you
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/ef5004f8-70f6-48ab-b543-aa2e9b0aacfe)
1. Add a comma preceding "rather than".
2. Add "used" preceding "in version 1.1.0".